site/content/en/docs/formats/celeba.md

```diff
@@ -0,0 +1,131 @@
+---
+title: 'CelebA'
+linkTitle: 'CelebA'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original CelebA dataset is available
+[here](https://drive.google.com/drive/folders/0B7EVK8r0v71pWEZsZE9oNnFzTm8?resourcekey=0-5BR16BdXnb8hVj6CNHKzLg).
```

Comment:
I think you should link the dataset's [homepage](https://mmlab.ie.cuhk.edu.hk/projects/CelebA.html) instead, since the authors might move the files to a different host later.